config/admins.txt

```diff
@@ -118,4 +118,3 @@ PKPenguin321 = Game Master
 TechnoAlchemist = Game Master
 Aloraydrel = Game Master
 
-
```

Comment:
no need to remove, also if you host with dream deamon rather then run inside dream maker, it will forcefully keep you admin while connected from the same machine, even let you bypass bans and bypass guest protection